src/polygon/src/polygon.cpp

```diff
@@ -15,8 +15,16 @@
 #include "polygon.h"
 
 #include <algorithm>
+#if MANIFOLD_PAR == 'T'
+#include <execution>
+#endif
 #include <list>
 #include <map>
+#if _APPLE_
+#include <experimental/memory_resource>
```

Comment:
However, it looks like apple doesn't even has complete `pmr` support in experimental: https://discourse.llvm.org/t/std-pmr-maturity/62200